siriuspy/siriuspy/devices/fofb.py

```diff
@@ -1549,31 +1547,13 @@ def fofbacc_decimation_value(self, value):
         self['FOFBAccDecimation-SP'] = value
 
     @property
-    def fofbacc_filter_enumvalue(self):
-        """Accumulator filter enum value."""
-        return self['FOFBAccFilter-Sts']
+    def psconfig_mat(self):
+        """Power Supply Configuration matrix."""
+        return self['PSConfigMat-RB']
 
-    @fofbacc_filter_enumvalue.setter
-    def fofbacc_filter_enumvalue(self, value):
-        self['FOFBAccFilter-Sel'] = value
-
-    @property
-    def fofbacc_filter_value(self):
-        """Accumulator filter value."""
-        return self['FOFBAccFilter-RB']
-
-    @fofbacc_filter_value.setter
-    def fofbacc_filter_value(self, value):
-        self['FOFBAccFilter-SP'] = value
-
-    @property
-    def fofbacc_filter_gain(self):
-        """Accumulator filter gain value."""
-        return self['FOFBAccFilterGain-RB']
-
-    @fofbacc_filter_gain.setter
-    def fofbacc_filter_gain(self, value):
-        self['FOFBAccFilterGain-SP'] = value
+    @psconfig_mat.setter
+    def psconfig_mat(self, value):
+        self['PSConfigMat-SP'] = value
```

Comment:
Perhaps it would be good to add some kind of checking here, for instance check if the shape of the array in ```value``` matches with the readback shape.